jx upgrade platform results in a SIGSEGV

Upgraded the cli from 1.3.420 to 1.3.459 and then did a `jx upgrade platform`. It then gave the output

```
Using helmBinary helm with feature flag: none
Current configuration dir: /Users/ericalford/.jx
options.Flags.CloudEnvRepository: https://github.com/jenkins-x/cloud-environments
options.Flags.LocalCloudEnvironment: false
Cloning the Jenkins X cloud environments repo to /Users/ericalford/.jx/cloud-environments
```

It then asked me

`? A local Jenkins X cloud environments repository already exists, recreate with latest? Yes`

then the output

```
Current configuration dir: /Users/ericalford/.jx
options.Flags.CloudEnvRepository: https://github.com/jenkins-x/cloud-environments
options.Flags.LocalCloudEnvironment: false
Cloning the Jenkins X cloud environments repo to /Users/ericalford/.jx/cloud-environments
Enumerating objects: 38, done.
Counting objects: 100% (38/38), done.
Compressing objects: 100% (30/30), done.
Total 1051 (delta 17), reused 20 (delta 8), pack-reused 1013
Using provider aws from team settings
Lets set up a Git username and API token to be able to perform CI/CD
```

then it asked me

`? Do you wish to use edalford11 as the GitHub username for CI/CD pipelines: Yes`

then it spit out this

```bash
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x2a38352]

goroutine 1 [running]:
github.com/jenkins-x/jx/pkg/jx/cmd.(*UpgradePlatformOptions).Run(0xc000a58000, 0x0, 0x0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/upgrade_platform.go:160 +0x492
github.com/jenkins-x/jx/pkg/jx/cmd.NewCmdUpgradePlatform.func1(0xc000a4ec80, 0x4e71930, 0x0, 0x0)
	/home/jenkins/go/src/github.com/jenkins-x/jx/pkg/jx/cmd/upgrade_platform.go:69 +0x66
github.com/spf13/cobra.(*Command).execute(0xc000a4ec80, 0x4e71930, 0x0, 0x0, 0xc000a4ec80, 0x4e71930)
	/home/jenkins/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:766 +0x2cc
github.com/spf13/cobra.(*Command).ExecuteC(0xc00002e000, 0xc0007e7b80, 0xc000878780, 0xc000864f00)
	/home/jenkins/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852 +0x2fd
github.com/spf13/cobra.(*Command).Execute(0xc00002e000, 0xc0007b6690, 0x35f2540)
	/home/jenkins/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800 +0x2b
github.com/jenkins-x/jx/cmd/jx/app.Run(0xc000943f88, 0xc000058118)
	/home/jenkins/go/src/github.com/jenkins-x/jx/cmd/jx/app/jx.go:17 +0x97
main.main()
	/home/jenkins/go/src/github.com/jenkins-x/jx/cmd/jx/jx.go:10 +0x22
```

--- a/pkg/jx/cmd/install.go
+++ b/pkg/jx/cmd/install.go
@@ -1061,7 +1061,7 @@ func (options *InstallOptions) cloneJXCloudEnvironmentsRepo() (string, error) {
 		return wrkDir, util.CopyDir(currentDir, wrkDir, true)
 	}
 	if options.Flags.CloudEnvRepository == "" {
-		return wrkDir, fmt.Errorf("No cloud environment git URL")
+		options.Flags.CloudEnvRepository = DEFAULT_CLOUD_ENVIRONMENTS_URL
 	}
 	log.Infof("Cloning the Jenkins X cloud environments repo to %s\n", wrkDir)
 	_, err = git.PlainClone(wrkDir, false, &git.CloneOptions{

--- a/pkg/jx/cmd/upgrade_platform.go
+++ b/pkg/jx/cmd/upgrade_platform.go
@@ -4,6 +4,11 @@ import (
 	"io"
 	"strings"
 
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
 	"github.com/jenkins-x/jx/pkg/helm"
 	"github.com/jenkins-x/jx/pkg/jx/cmd/templates"
 	"github.com/jenkins-x/jx/pkg/log"
@@ -11,10 +16,6 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"gopkg.in/AlecAivazis/survey.v1/terminal"
-	"fmt"
-	"path/filepath"
-	"os"
-	"io/ioutil"
 	core_v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
@@ -157,7 +158,8 @@ func (o *UpgradePlatformOptions) Run() error {
 	}
 
 	helmConfig := &o.CreateEnvOptions.HelmValuesConfig
-	if helmConfig.ExposeController.Config.Domain == "" {
+	exposeController := helmConfig.ExposeController
+	if exposeController != nil && exposeController.Config.Domain == "" {
 		helmConfig.ExposeController.Config.Domain = o.InitOptions.Flags.Domain
 	}
 

